libs/deal-ticket/src/components/deal-ticket/deal-ticket-price-tp-sl.tsx

```diff
@@ -0,0 +1,172 @@
+import { Controller, type Control } from 'react-hook-form';
+import type { Market } from '@vegaprotocol/markets';
+import type { OrderFormValues } from '../../hooks/use-form-values';
+import { toDecimal, useValidateAmount } from '@vegaprotocol/utils';
+import {
+  TradingFormGroup,
+  TradingInputError,
+  Tooltip,
+  FormGroup,
+  Input,
+  InputError,
+  Pill,
+} from '@vegaprotocol/ui-toolkit';
+import { useT } from '../../use-t';
+
+export interface DealTicketPriceTakeProfitStopLossProps {
+  control: Control<OrderFormValues>;
+  market: Market;
+  takeProfitError?: string;
+  stopLossError?: string;
+  quoteName?: string;
+}
+
+export const DealTicketPriceTakeProfitStopLoss = ({
+  control,
+  market,
+  takeProfitError,
+  stopLossError,
+  quoteName,
+}: DealTicketPriceTakeProfitStopLossProps) => {
+  const t = useT();
```

Comment:
It looks like none of labels used in DealTicketPriceTakeProfitStopLoss were added to i18n/src/locales/en/deal-ticket.json